src/component/app.jsx

```diff
@@ -167,64 +153,31 @@ export default class App extends Component {
                             <img
                                 src="public/logo.png"
                                 width="32"
                                 height="32"
                                 className={styles.drawerTitleLogo}
                             />
-                            <span className={styles.drawerTitleText}>
-                                Unleash
-                            </span>
+                            <span className={styles.drawerTitleText}>Unleash</span>
                         </span>
                         <hr />
                         <Navigation className={styles.navigation}>
-                            {createListItem(
-                                '/features',
-                                'Feature Toggles',
-                                'list',
-                                true
-                            )}
-                            {createListItem(
-                                '/strategies',
-                                'Strategies',
-                                'extension',
-                                true
-                            )}
-                            {createListItem(
-                                '/history',
-                                'Event History',
-                                'history',
-                                true
-                            )}
-                            {createListItem(
-                                '/archive',
-                                'Archived Toggles',
-                                'archive',
-                                true
-                            )}
-                            {createListItem(
-                                '/applications',
-                                'Applications',
-                                'apps',
-                                true
-                            )}
+                            {createListItem('/features', 'Feature Toggles', 'list', true)}
+                            {createListItem('/strategies', 'Strategies', 'extension', true)}
+                            {createListItem('/history', 'Event History', 'history', true)}
+                            {createListItem('/archive', 'Archived Toggles', 'archive', true)}
+                            {createListItem('/applications', 'Applications', 'apps', true)}
                         </Navigation>
                         <hr />
                         <Navigation className={styles.navigation}>
                             <a
                                 href="https://github.com/Unleash"
                                 target="_blank"
-                                className={[
-                                    styles.navigationLink,
-                                    'mdl-color-text--grey-900',
-                                ].join(' ')}
+                                className={[styles.navigationLink, 'mdl-color-text--grey-900'].join(
```

Comment:
These should just be 
```js
className={`${styles.navigationLink} ${mdl-color-text--grey-900}`}
```

No idea why they are in an array and `join`ed

Reply:
I would say:

```jsx
className=`${styles.navigationLink} mdl-color-text--grey-900`
```

Reply:
haha, of course! Was a bit quick there!